libraries/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -287,6 +301,14 @@ export class RushConfiguration {
    */
   public readonly commonTempFolder: string;
 
+  /**
+   * The folder where temporary files will be stored for subspaces. The specific folder will
+   * append the subspace name to this path.
+   *
+   * Example: `C:\MyRepo\common\temp\<subspace_name>`
```

Comment:
When this is initialized on line 647, it looks like it's set to just `<repo root>/common/temp`.

Is this property useful to have around, since it appears to actually just be the same as `commonTempFolder`?

Reply:
I guess this is not necessary right now, so removed